src/sentry/migrations/0171_backfill_cool.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 1.11.29 on 2021-03-11 06:29
+
+from django.db import migrations
+
+
+def backfill_is_cool(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+
+    for user in User.objects.all():
+        is_cool = user.id % 1337 == 0
+        user.update(is_cool=is_cool)
```

Comment:
`update` is a function in the [sentry codebase](https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/query.py#L15), and generally we want to avoid using functions from our codebase in migrations since they can be removed or changed, this means that onprem users updating might fail to run this migration if we removed `update` for example.

if we really want to use `update` here for whatever reason, we should vendor the entire update function at the time of this migration instead.

But in this specific case, it might be easier to just use django's `save` function instead

```suggestion
        user.is_cool = user.id % 1337 == 0
        user.save()
```

Also might be able to save some db ops if we only run `save` if its changed from what's currently saved in the db